.github/workflows/publish-website.yml

```diff
@@ -46,6 +46,15 @@ jobs:
           deriv_me_site_id: ${{ secrets.DERIV_ME_SITE_ID }}
           deriv_be_site_id: ${{ secrets.DERIV_BE_SITE_ID }}
           webflow_general_site_id: ${{ secrets.WEBFLOW_GENERAL_SITE_ID }}
+      
+      - name: Trigger Scraper workflow
+        id: trigger-scraper
```

Comment:
we need to add a if condition here to make sure that we are triggering it only when we are doing a production release.
Basically when we are releasing on deriv.com and deriv.be and deriv.me